### ISSUE TITLE
Rename PPPL sub-community

### ISSUE THREAD

**Issue by astrochun** [pulibrary/pdc_describe#1663]
Per @kelynch, the following PPPL sub-community needs to be renamed:
 - "ITER and Tokamaks" -> "Tokamak Experimental Sciences"

The renaming will assist in the final migration of DataSpace collection ARKs.

Acceptance criteria:

- [ ] Change the code locations of "ITER and Tokamaks" to "Tokamak Experimental Sciences" in the Group Model
- [ ] Create a migration (maybe database) to update existing database records "ITER and Tokamaks" to "Tokamak Experimental Sciences"

**Comment by carolyncole** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/RDSS-61a4f1a12a399b001730f65a/issues/pulibrary/pdc_describe/1663?planning-poker) with Zenhub @hectorcorrea @JaymeeH @jrgriffiniii @leefaisonr

**Comment by astrochun** [comment]
@kelynch @hectorcorrea @jrgriffiniii 

Upon further investigation, I realized recently that what PPPL most likely want to do is to still have a separate ITER sub-community. Namely that "Tokamak Experimental Sciences" should be a new one we add. That's because TES has multiple areas of work, including NSTX-U under the new reorganization. ITER is another area and it's anticipated to grow so having work distinguished as ITER may in fact be fruitful.

I've modified the original request and I'm updating #1662
